### PR TITLE
Changes to just get Devices with given DIDs

### PR DESCRIPTION
**Pull request by DerKarussellbremser**
For Sure, there is a littlebit to do.
An option to enable and disable this feature.
An easy method to get the DIDs from the devices with all names so you don't need to guess the right one.

### CODE REVIEW

**Comment by bombadiltom** [comment]
Thanks for this great idea, but without these features I cannot merge this, because everyone would have to add all his DIDs:

# To-Do
* Option to enable and disable the DID Feature
* option to choose if you include or exclude the given DIDs

At least check if the list is empty or not existing at all and then everything must work as before.

File: index.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function RademacherHomePilot(log, config, api) {
                     body.devices.forEach(function(data) {
                         var uuid = UUIDGen.generate("did"+data.did);
                         var accessory = self.accessories[uuid];
-                        
+                    if ((config["DIDs"]).includes(data.did)) {
```
Comment (bombadiltom) [inline index.js:53]: add check for non-existing variable and empty list

File: index.js [hunk 2]
```diff
@@ -142,6 +147,8 @@ function RademacherHomePilot(log, config, api) {
                     body.meters.forEach(function(data) {
                         var uuid = UUIDGen.generate("did"+data.did);
                         var accessory = self.accessories[uuid];
+
+                    if ((config["DIDs"]).includes(data.did)) {
```
Comment (bombadiltom) [inline index.js:151]: add check for non-existing variable and empty list

Comment (DerKarussellbremser) [reply index.js]: Inserted the DID as Option also changed the files around and made a check if the DIDs are set when want to use the feature.
Really not sure if its the best way how i inserted it .. i'm more comfortable with assembler, this is my first try to change a javascript in such a way with inserting new features.

**Pull request merged**